### PR TITLE
Prac 05 feedback

### PR DESCRIPTION
**Pull request by Rinnki97**
@Liu1xuan
@AlexZhang21
@Byeclouds

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -2,17 +2,24 @@
 CP1404/CP5632 Practical
 State names in a dictionary
 File needs reformatting
+
+Estimate: 5 minutes
+Actual:   5 minutes
+
 """
 
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
```
Comment (Byeclouds) [inline prac_05/state_names.py]: i think delete the TODO message will make the code looks better

File: prac_05/state_names.py [hunk 2]
```diff
@@ -2,17 +2,24 @@
...
-CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
-            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
 
-state_code = input("Enter short state: ")
```
Comment (Byeclouds) [inline prac_05/state_names.py]: good comment for code

File: prac_05/wimbledon.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        for line in in_file:
+            row = line.strip().split(',')
+            data.append(row)
+    return data
+
+
+def get_champions(data):
+    champions = {}
+    for row in data:
+        champion = row[2]
```
Comment (Byeclouds) [inline prac_05/wimbledon.py]: better to define magic number as a constant outside the function

File: prac_05/state_names.py (hunk 1 above)
Comment (Liu1xuan) [reply prac_05/state_names.py]: I think the work be finished well.

File: prac_05/state_names.py (hunk 2 above)
Comment (Liu1xuan) [reply prac_05/state_names.py]: nice comment

File: prac_05/wimbledon.py (hunk 3 above)
Comment (Liu1xuan) [reply prac_05/wimbledon.py]: the magic number should ideally be defined as a constant outside of the function.

Comment (AlexZhang21) [reply prac_05/wimbledon.py]: Define constants outside of functions

File: prac_05/state_names.py (hunk 2 above)
Comment (AlexZhang21) [reply prac_05/state_names.py]: perfect job

**Pull request merged**